.demisto-sdk-report

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
please remove this file from the PR (it will be [added to `.gitignore`](https://github.com/demisto/demisto-sdk/pull/1947) soon)